src/main/java/com/suse/saltstack/netapi/event/EventStream.java

```diff
@@ -64,13 +72,11 @@ public EventStream() {
      *
      * @param config Contains the necessary details such as EndPoint URL and
      * authentication token required to create the WebSocket.
+     * @throws SaltStackException in case of an error during stream initialization
      */
-    public EventStream(ClientConfig config) {
-        try {
-            initializeStream(config);
-        } catch (SaltStackException e) {
-            e.printStackTrace();
-        }
+    public EventStream(ClientConfig config) throws SaltStackException {
+        maxMessageLength = config.get(ClientConfig.WEBSOCKET_MAX_MESSAGE_LENGTH);
```

Comment:
@AdamMajer  Not really. The whole mystery of the `maxMessageLength = 0` or `-1` etc, that it means to disable the very message length check. Consequentially, your code would invalidate this feature. Also default value is taken properly here: https://github.com/SUSE/saltstack-netapi-client-java/blob/master-support-partial-websocket-messages/src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java#L108